### PR TITLE
continue to continue 2

### PR DESCRIPTION
**Pull request by gncdev**
fix

### CODE REVIEW

**Comment by dktapps** [comment]
what exactly does this fix?

File: src/aliuly/manyworlds/MwLvDat.php [hunk 1]
```diff
@@ -127,15 +127,16 @@ public function onSCommand(CommandSender $c, Command $cc, $scmd, $data, array $a
...
 					}
 					$changed = true;
 					$unload = true;
 					$provider->getLevelData()->setString("generatorOptions", $v);
 					break;
 				default:
 					$c->sendMessage(mc::_("Unknown key %1%, ignored", $k));
-					continue;
+					continue 2;
+					break;
```
Comment (dktapps) [inline src/aliuly/manyworlds/MwLvDat.php:139]: this is useless

**Comment by gncdev** [comment]
this code fix's:
on command not response fix
and continue; fix

**Comment by gncdev** [comment]
plugin works all function

**Pull request closed without merging**